release-please-config.json

```diff
@@ -0,0 +1,13 @@
+{
+  "packages": {
```

Comment:
I'm curious - why we don't need these config files for passage-node as well?

Reply:
we do -- it was done in https://github.com/passageidentity/passage-node/pull/181

Reply:
ah I missed that PR